models/Position.ts

```diff
@@ -0,0 +1,17 @@
+import { Position } from "@/queries/conditional-tokens/types";
+
+export class PositionModel {
+  conditionId: string;
+  outcomeIndex: number;
+  outcomes: string[] | undefined | null;
```

Comment:
I think we shouldn't have undefined here. If there aren't outcomes, is null, otherwise is a string array.

Reply:
but then the type will shout cause, if positions has no conditions the "?" will return undefined.

Reply:
but we could counter that, but also position.conditions[0].outcomes can come as undefined. So we would have to handle that case, to make it null. Seems not to be worth it.

Reply:
> but then the type will shout cause, if positions has no conditions the "?" will return undefined.

So we can remove the null in this case?